tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/palm_detection.py

```diff
@@ -0,0 +1,255 @@
+        scales = self.get_value_from_config('scales')
+        assert len(scales) == 4
+        self.x_scale, self.y_scale, self.w_scale, self.h_scale = scales
+        self.min_score_thresh = self.get_value_from_config('min_score_thresh')
+        self.apply_exp_on_box_size = self.get_value_from_config('apply_exp_on_box_size')
+        self.num_classes = self.get_value_from_config('num_classes')
+
+    def select_output_blob(self, outputs):
+        self.scores_out = self.check_output_name(self.scores_out, outputs)
+        self.boxes_out = self.check_output_name(self.boxes_out, outputs)
+        self.outputs_verified = True
+
+    def process(self, raw, identifiers, frame_meta):
+        result = []
+        raw_output = self._extract_predictions(raw, frame_meta)
+        if not self.outputs_verified:
+            self.select_output_blob(raw_output)
+
+        for identifier, raw_scores, raw_boxes in zip(identifiers, raw_output[self.scores_out],
+                                                     raw_output[self.boxes_out]):
+            num_boxes, _ = raw_boxes.shape
+            boxes = self.decode_boxes(raw_boxes)
+            detection_scores = np.zeros(num_boxes)
+            detection_classes = np.zeros(num_boxes)
+
+            for i in range(num_boxes):
+                class_id = -1
+                max_score = -np.inf
+                for score_idx in range(self.num_classes):
+                    score = raw_scores[i, score_idx]
+                    if self.sigmoid_score:
+                        if self.score_clipping_thresh:
+                            score = np.clip(score, (-1) * self.score_clipping_thresh, self.score_clipping_thresh)
+                        score = 1 / (1 + np.exp((-1) * score))
+                    if max_score < score:
+                        max_score = score
+                        class_id = score_idx
+                detection_classes[i] = class_id
+                detection_scores[i] = max_score
+            cond = detection_scores >= self.min_score_thresh
+            boxes = np.array(boxes)[cond]
+            detection_classes = detection_classes[cond]
+            detection_scores = detection_scores[cond]
+
+            cond = ((boxes[:, 2] - boxes[:, 0]) >= 0) & ((boxes[:, 3] - boxes[:, 1]) >= 0)
+
+            boxes = boxes[cond, :]
+            detection_classes = detection_classes[cond]
+            detection_scores = detection_scores[cond]
+
+            num_boxes, _ = boxes.shape
+
+            x_mins = []
+            y_mins = []
+            x_maxs = []
+            y_maxs = []
+            labels = []
+            det_scores = []
+            for i in range(num_boxes):
+                x_mins.append(boxes[i, 1])
+                y_mins.append(boxes[i, 0])
+                x_maxs.append(boxes[i, 3])
+                y_maxs.append(boxes[i, 2])
+                labels.append(detection_classes[i])
+                det_scores.append(detection_scores[i])
```

Comment:
as I can see moving detection_classes and detection_scores is useless, you can use them as is
Regarding boxes, it can be simplified
```suggestion
            x_mins, y_mins, x_maxs, y_maxs = boxes.T
```

Reply:
Boxes shape is [num_boxes, 18], where 18 is 4 points of x_min, y_min, x_max, y_max, and remaining 14 is 7 pairs of keypoints coordinates. In mediapipe code keypoints coordinates are calculated but never used. Should I remove keypoints from boxes data?

Reply:
fixed